docker-compose.yml

```diff
@@ -13,9 +13,7 @@ services:
     environment:
       BROWSER_STACK_USERNAME:
       BROWSER_STACK_ACCESS_KEY:
-      SAUCE_LABS_USERNAME:
```

Comment:
We removed support for Sauce Labs in Maze Runner some time ago and do not have an account with them in any case.